### PR TITLE
Added a CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by YashdalfTheGray**
Contributing to flintandsteel in style! Refers to issue #79!

Reviewers requested: @Mctalian @bcbrennecke @davethenipper @LongArmMcGee

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+### Fork flintandsteel and set up your repo
+Click the fork button on our repository page to get a copy for yourself. You can then create a local clone of your fork using `git clone <repo_name>`. You can now begin making your changes to the code. 
+
+Refer to [README.md](./README.md) for installation and execution instructions. 
+
+### Make your changes
+
+After you make your changes and before committing, make sure to
+* run `gulp jshint` to lint the code and solve any issues presented
```
Comment (Mctalian) [inline CONTRIBUTING.md]: Should we also include `test:client` here as a requirement?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+## Submission Guidelines
+
+We use the forking methodology for contributions. This enables us to keep the code being commited under check to guarantee code quality. It also enables members to perform code reviews without the need for another tool. All the development on the project and the current state of the project are maintained on the `master` branch. 
+
+We use [Trello](https://trello.com/) to track our development. If you would like to become a regular contributor to this project, please let the team know and we can discuss further. 
+
+Contribute to our project by following these steps
+
+1. Fork flintandsteel and clone it locally
```
Comment (Mctalian) [inline CONTRIBUTING.md]: I'm not sure how to include it, but perhaps a reminder to rebase to latest before making changes if someone has already forked/cloned the repo.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+Pull requests changing significant portions of code without updating the unit tests will be rejected and asked to be redone.
+
+### Create a Pull Request
+
+After you have pushed your changes to your fork, please create a pull request from your fork back to our repository. Make sure that the pull request details the changes that you will be merging and that your commits have meaningful comments. 
+
+Also make sure that you ask at least 2 members to review your code. Once both members review and approve the code, we will merge the pull request. Once the code reviews are requested, members are expected to respond within 2 days and are expected to leave constructive feedback, if required. 
+
+Once the pull request is merged, make sure that you update your master before begining further work on your flintandsteel fork.
```
Comment (Mctalian) [inline CONTRIBUTING.md]: Small typo: ~~begining~~ -> beginning

File: README.md [hunk 4]
```diff
@@ -38,11 +38,9 @@ To start with some predefined data, `gulp generate:data` populates the datastore
...
 ## Contributing
 
-Before committing code, make sure
-* you run `gulp jshint` to lint the code and solve any issues presented
-* change `package.json` and `bower.json` if a new module was installed using the `--save` command line switch, i.e. `[npm|bower] install --save {module_name}`
-* Add `package.json` or `bower.json` to a separate commit with a reason to pull that module into the build
-* Update the unit tests in the corresponding .spec.js file in the same folder. If one doesn't exist, create a file with the same name as the folder followed by ".spec.js". This marks the file as a unit test file.
+Refer to [CONTRIBUTING.md](./CONTRIBUTING.md) for directions on how to set up and contribute to this project. 
+
+Constributions using other channels will be disregarded and asked to be routed through the proper procedure.
```
Comment (Mctalian) [inline README.md]: Typo: ~~Constributions~~ -> Contributions

**Comment by Mctalian** [comment]
:+1:

File: CONTRIBUTING.md (hunk 2 above)
Comment (YashdalfTheGray) [inline CONTRIBUTING.md]: That's a good idea, I forgot about that.

File: CONTRIBUTING.md (hunk 1 above)
Comment (YashdalfTheGray) [inline CONTRIBUTING.md]: Definitely.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+1. Fork flintandsteel and clone it locally
+2. Make your changes and push to your fork
+3. Create a Pull Request back to our repository
+
+### Fork flintandsteel and set up your repo
+Click the fork button on our repository page to get a copy for yourself. You can then create a local clone of your fork using `git clone <repo_name>`. You can now begin making your changes to the code. 
+
+**NOTE**: If you have already forked the repo and cloned it, make sure to rebase before starting to make changes. This will ensure that you have the latest code. 
+
```
Comment (YashdalfTheGray) [inline CONTRIBUTING.md]: @Mctalian I added a note here about the rebase thing.

File: CONTRIBUTING.md (hunk 3 above)
Comment (YashdalfTheGray) [inline CONTRIBUTING.md]: Changed.

File: README.md (hunk 4 above)
Comment (YashdalfTheGray) [inline README.md]: Changed.

**Comment by davethenipper** [comment]
Looks great.  I'll try things out this weekend and see if I have any feedback

**Comment by YashdalfTheGray** [comment]
So can I merge the PR or are you deferring your review until this weekend?

**Comment by davethenipper** [comment]
Merge approved

**Pull request merged**